pkg/binary/envoy/debug/node_test.go

```diff
@@ -41,3 +41,21 @@ func Test_ps(t *testing.T) {
 		}
 	})
 }
+
+func Test_network_inerfaces(t *testing.T) {
```

Comment:
This test should be integrated with the one above as they do the same things. Add a table-driven test for each file and use only a single run.

Reply:
Right 👍